src/xdocs/config_header.xml

```diff
@@ -63,7 +63,7 @@ line 5: ////////////////////////////////////////////////////////////////////
           <tr>
             <td>headerFile</td>
             <td>name of the file containing the required header</td>
-            <td><a href="property_types.html#string">string</a></td>
+            <td><a href="property_types.html#string">String</a></td>
```

Comment:
I think it is reasonable to make new type URI , like we have http://checkstyle.sourceforge.net/property_types.html#regexp and explain that it is in string form. All types are in string form in config, but we do treat them in special way that is why we have special description of type on our site.